src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/SamlNameIdentifierClaimResource.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace CoreWCF.IdentityModel.Tokens
 {
-    internal class SamlNameIdentifierClaimResource
+    public class SamlNameIdentifierClaimResource
```

Comment:
If we're making this class public, it needs to match .NET Framework for serialization. There's a missing `[DataContract]` attribute, and the fields marked with `[DataMember]` need to either revert to having the same names as .NET Framework, or set the `Name` property on the attribute. Otherwise it will serialize differently and won't be compatible if someone is serializing it with DataContractSerializer.